src/walkthrough/sample_scene.py

```diff
@@ -0,0 +1,43 @@
+from manimlib import *
+
+# ❗
+# ❗ First, make sure to save this file somewhere. Otherwise, nothing will work here.
+# ❗
+
+
+class MyFirstManimNotebook(Scene):
+    def construct(self):
+        ## Your first Manim Cell
+        # Note how a Manim Cell is introduced by a comment starting with `##`.
+        # You should see a button `Preview Manim Cell` above this cell.
+        # Click on it to preview the animation.
+        circle = Circle()
+        circle.set_stroke(BLUE_E, width=4)
+        self.play(ShowCreation(circle))
+
+        ## Transform circle to square
+        # You can also preview the cell with the default hotkey:
+        # `Cmd+' Cmd+e` (MacOS) or `Ctrl+' Ctrl+e` (Windows/Linux)
+        square = Square()
+        self.play(ReplacementTransform(circle, square))
+        self.wait(1.5)
+
+        ## Make it red and fly away
+        self.play(
+            square.animate.set_fill(RED_D, opacity=0.5),
+            self.camera_config.frame.animate.set_width(25),
```

Comment:
bhoov wrote:

> FYI, I'm getting a bug in the 3rd cell "Make it red and fly away". AttributeError: 'dict' object has no attribute 'frame', which could be fixed by instead calling self.frame.animate.set_width(25). I'm guessing this is a breaking change in my manimgl==v1.7.1

Reply:
Mmh, good point, will check that... At the moment, with the latest Manim commit 3h ago (bafea89ac9d11dc168bce923a65501ec74312afe) and the latest VSCode version `1.96.0`, previewing doesn't work for me at all, i.e. the cells are not copied to the clipboard anymore. Will investigate in the next days...

Reply:
I figured that my clipboard timeout setting of `850ms` was not sufficient anymore.  And it feels like the `checkpoint_paste()` command takes longer now to start which could be linked to recent refactorings that 3b1b made. Do you experience the same behavior?

Reply:
Or maybe that's just a figment. I've just checked out several commits of Manim of the past and for all of them it takes a similar amount of time to print anything to the console after a `checkpoint_paste()`. Maybe it's just that I've worked too long on the `reload()` command which is almost instantaneous for me ;)

Reply:
Oh weird, the example is working totally fine for me. 

ManimGL v1.7.1
VSCode v1.95.0